File: Golang/3224.go
package main

import "fmt"

func main() {
	var jon, doc string
	fmt.Scanln(&jon)
	fmt.Scanln(&doc)

	if len(jon) >= len(doc) {
		fmt.Println("go")
	} else {
		fmt.Println("no")
	}
}

File: 1. Beginner/3224/3224.scala
object Main {
    def main(args: Array[String]) {
        val jon: String = scala.io.StdIn.readLine()
        val doc: String = scala.io.StdIn.readLine()
        
        if (jon.length >= doc.length) {
            println("go")
        } else {
            println("no")
        }
    }
}